src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -33,14 +26,964 @@ to make sure IE does not go into compatibility mode.
+an exception it can be captured.  For some SDKs you can also omit the argument to
+`capture_exception` and it will attempt to capture the current exception.
+
+```
+try {
+  aFunctionThatMightFail();
+} catch (err) {
+  Sentry.captureException(err);
+}
+```
+&nbsp;
+### Generate a Custom Error
+**[Mimi note: I'd like to flesh this out more, but need help.]**
+
+You can generate a custom error with `new Error('message')`.
+
+&nbsp;
+### Setting Extra Context
+In addition to the [structured context](#context) that Sentry understands, you can send arbitrary key/value pairs of data which the Sentry SDK will store alongside the event. These are not indexed, and the Sentry SDK uses them to add additional information about what might be happening:
+
+```
+Sentry.configureScope((scope) => {
+  scope.setExtra("character_name", "Mighty Fighter");
+});
+```
+
+{% capture __alert_content -%}
+**Be aware of maximum payload size** - There are times, when you may want to send the whole application state as extra data. Sentry does not recommend this, as application state can be very large and easily exceed the 200kB maximum that Sentry has on individual event payloads. When this happens, you'll get an `HTTP Error 413 Payload Too Large` message as the server response or (when you set `keepalive: true` as a `fetch` parameter), the request will stay `pending` forever (e.g. in Chrome).
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Note"
+  content=__alert_content
+  level="warning"
+%}
+
+&nbsp;
+### Capturing Messages
+
+Another common operation is to capture a bare message.  A message is just some textual
+information that should be sent to Sentry.  Typically messages are not emitted but
+there are situations when this is useful.
+
+```
+Sentry.captureMessage('Something went wrong');
+```
+
+&nbsp;
+### Capturing Events
+
+SDKs also provide ways to capture entire custom event objects. This is what integrations internally use to capture bespoke events that the SDK supplies with a lot of extra data.
+
+&nbsp;
+## Context
+You can also set context when manually triggering events.
+
+### Setting Context {#context}
+Sentry supports additional context with events. Often this context is shared among any issue captured in its lifecycle, and includes the following components:
+
+**Structured Contexts**
+
+: Specific structured contexts --- OS info, runtime information, etc.  This is typically set automatically.
+
+[**User**](#capturing-the-user)
+
+: Information about the current actor
+
+[**Tags**](#tagging-events)
+
+: Key/value pairs which generate breakdown charts and search filters
+
+[**Level**](#setting-the-level)
+
+: An event's severity 
+
+[**Fingerprint**](#setting-the-fingerprint)
+
+: A value used for grouping events into issues
+
+[**Unstructured Extra Data**](#extra-context)
+
+: Arbitrary unstructured data which the Sentry SDK stores with an event sample
+
+&nbsp;
+### Extra Context
```

Comment:
Is this a purposeful duplication of **Setting Extra Context** from a few paragraphs above?

Reply:
@ckj Yeah, I'm worried a dev would skip the Manually Trigger Errors section and miss the "Setting Extra Context" section. I'll just have line 215 point to the "Setting Extra Context" above.

Reply:
Okay. Made the change. :)